phpcs.xml.dist

```diff
@@ -1,16 +1,12 @@
 <?xml version="1.0"?>
-<ruleset name="WPP-PerfLab">
-	<description>WordPress Coding Standards for the Performance Lab Plugin</description>
+<ruleset name="WPP">
+	<description>WordPress Coding Standards for the Performance Features.</description>
 	<rule ref="bin/phpcs/phpcs.ruleset.xml"/>
 
 	<config name="text_domain" value="performance-lab,default"/>
 
-	<rule ref="WordPress.Files.FileName.NotHyphenatedLowercase">
-		<exclude-pattern>includes/server-timing/object-cache.copy.php</exclude-pattern>
-	</rule>
-
 	<file>.</file>
-	<exclude-pattern>./node_modules/*</exclude-pattern>
+
+	<!--Ignore plugins directory-->
 	<exclude-pattern>./plugins/*</exclude-pattern>
-	<exclude-pattern>./vendor/*</exclude-pattern>
```

Comment:
The `node_modules` and `vendor` directories are removed because they are excluded by default?

Reply:
Nevermind. They were added to `phpcs.ruleset.xml`

Reply:
They are excluded in the global ruleset.

https://github.com/WordPress/performance/blob/d9cd4d4d6e47de59e301228399c8fe8e400cf873/bin/phpcs/phpcs.ruleset.xml#L136-L137